extensions/amp-story/0.1/amp-story-desktop.css

```diff
@@ -62,20 +63,22 @@ amp-story[standalone][desktop] {
 
 .i-amphtml-story-background {
   position: absolute !important;
-  top: 0 !important;
-  right: 0 !important;
-  bottom: 0 !important;
-  left: 0 !important;
+  top: -150px !important;
+  right: -150px !important;
+  bottom: -150px !important;
+  left: -150px !important;
   opacity: 0 !important;
   filter: blur(50px) !important;
-  margin: -60px !important;
   background-size: cover !important;
   background-color: transparent !important;
   background-position: center center !important;
+  will-change: opacity, z-index !important;
```

Comment:
are we trying to animate opacity?
last time we did this, opacity animation + page movement tend to get a little janky. please check

Reply:
Confirmed as working, issues were caused by lack of transition sync/two transitions at the same time.